examples/bbciplayer_example.py

```diff
@@ -0,0 +1,51 @@
+"""
+Example on how to use the BBC iPlayer Controller
+
+"""
+# pylint: disable=invalid-name
+
+import logging
+from time import sleep
+import sys
+
+import pychromecast
+from pychromecast.controllers.bbciplayer import BbcIplayerController
+
+# Change to the name of your Chromecast
+CAST_NAME = "Lounge Video"
+
+# Note: Media ID is NOT the 8 digit alpha-numeric in the URL
+# it can be found by right clicking the playing video on the web interface
+# e.g. https://www.bbc.co.uk/iplayer/episode/b09w7fd9/bitz-bob-series-1-1-castle-makeover shows: 
+# "2908kbps | dash (mf_cloudfront_dash_https)
+#  b09w70r2 | 960x540"
```

Comment:
This is a great explanation, add it in pychromecast/controllers/bbciplayer.py as well